### ISSUE TITLE
torch.floor_divide is deprecated

### ISSUE THREAD

**Issue by digital-idiot** [rusty1s/pytorch_scatter#236]
[torch.floor_divide](https://pytorch.org/docs/stable/generated/torch.floor_divide.html) is deprecated. Occurrences of `floor_divide` (example below) should be possibly replaced with `out.div(count, rounding_mode='floor')`.

https://github.com/rusty1s/pytorch_scatter/blob/711bc05ca2b4007293ce45f6bafb6a25612e128a/torch_scatter/scatter.py#L58

**Comment by rusty1s** [comment]
Yes, I‘m aware of this. Since we try to support the latest to versions of PyTorch, this will be fixed once PyTorch 1.10 is released.
